requirements.txt

```
@@ -2,5 +2,5 @@ numpy~=1.14
 scipy~=1.0
 scikit-learn>=0.19
 six~=1.10
-tensorflow>=1.12.1,<2
+tensorflow>=2.0.1
```

`muffnn` would require some work to switch it to use Tensorflow v2. Let's try the patched version of v1.
```suggestion
tensorflow>=1.15.2,<2
```

---

+1. [The latest build run on master](https://travis-ci.org/civisanalytics/muffnn/jobs/621643563?utm_medium=notification&utm_source=github_status) did use tensorflow v1.15.0, and so I'd imagine v1.15.2 _should_ be good with muffnn as-is.